src/NovelRT/Windowing/WindowingService.cpp

```diff
@@ -24,6 +24,22 @@ namespace NovelRT::Windowing {
     float wData = displayData->width * 0.7f;
     float hData = displayData->height * 0.7f;
 
+#if defined (_WIN64)
+    auto optimus = LoadLibrary("nvapi64.dll");
+    if (optimus != nullptr) {
+      _logger.logInfoLine("NVIDIA GPU detected. Enabling...");
+    } else {
+      _logger.logWarningLine("NVIDIA GPU not detected. Continuing w/o Optimus support.");
+    }
```

Comment:
Maybe this should be a method? You kinda just have this same snippet twice.

Reply:
Done.